apps/coscrad-frontend/src/utils/generic-components/presenters/multilingual-text-item-presenter.tsx

```diff
@@ -0,0 +1,27 @@
+import { IMultilingualTextItem } from '@coscrad/api-interfaces';
+import LanguageIcon from '@mui/icons-material/Language';
+import { IconButton, Tooltip, Typography } from '@mui/material';
+import { Variant } from '@mui/material/styles/createTypography';
+import { getLabelForLanguage } from './text-presenters/get-label-for-language';
+
+interface MultilingualTextItemPresenterProps {
+    variant: Variant;
```

Comment:
Yea- this could be a bit dangerous. What if you want to flow other style props through to Typography? Maybe what you really want is to just build the text with tooltip and then wrap it outside of this component with whatever you might choose to wrap it with. But I'll defer to you on this choice.